openedx/core/djangoapps/plugins/plugin_contexts.py

```diff
@@ -0,0 +1,81 @@
+from importlib import import_module
+
+from logging import getLogger
+
+from openedx.core.lib.cache_utils import process_cached
+
+from . import constants, registry
+
+
+log = getLogger(__name__)
+
+def get_plugins_view_context(project_type, view_name, existing_context={}):
+    """
```

Comment:
Since this is a low-level library function, it would be great to explain each of the parameters (project_type, view_name, ..) in the docstring.